jfrog-applications/jfrog-cli/cli-for-jfrog-security/scan-your-source-code.md

```diff
@@ -52,6 +52,7 @@ This command also supports the following Advanced Scans with the **Advanced Secu
 | --exclude-test-deps   | <p>[Default: false] [Gradle]<br><br>Set to true if you'd like to exclude Gradle test dependencies from Xray scanning.</p>                                                                                                                                                                                                                                         |
 | --requirements-file   | <p>[Optional] [Pip]<br><br>Defines pip requirements file name. For example: 'requirements.txt'</p>                                                                                                                                                                                                                                                                |
 | --working-dirs        | <p>[Optional]<br><br>A comma-separated list of relative working directories, to determine the audit targets locations.</p>                                                                                                                                                                                                                                        |
+| --exclusions        | <p>[Default: \*node_modules\*;\*target\*;\*venv\*;\*test\*]<br><br>List of exclusions separated by semicolons, utilized to skip sub-projects from undergoing an audit. These exclusions may incorporate the * and ? wildcards.</p>                                                                                                                                                                                                                                        |
```

Comment:
* This line seems to be misaligned with the lines of the other command options.
* Let's add an example (in the below examples section) for using the new option.